packages/create-block/package.json

```diff
@@ -20,8 +20,7 @@
 		"url": "https://github.com/WordPress/gutenberg/issues"
 	},
 	"engines": {
-		"node": ">=14",
-		"npm": ">=6.14.4"
```

Comment:
There is something specific with this npm version - probably it's where npm aliases were added. It made sense with Node 18 to remove it, for Node 14, it's better to keep it. We can remove it again when we upgrade Node 16 in ~2 months.